db/migrate/20200221144825_add_vssc_extract_date_drivetime_bands_default.rb

```diff
@@ -0,0 +1,5 @@
+class AddVsscExtractDateDrivetimeBandsDefault < ActiveRecord::Migration[5.2]
```

Comment:
Do these migrations need to be extracted to a separate PR?

Reply:
Because this code is not affecting the database schema at all (not to mention the fact that this code isn't triggered by requests but rather a sidekiq job) it can be applied after the code has been merged. It is mostly just a paranoid attempt at handling a possible reversion in the PSSG API.